suite-native/app/android/app/build.gradle

```diff
@@ -129,7 +129,7 @@ def enableProguardInReleaseBuilds = false
  * give correct results when using with locales other than en-US.  Note that
  * this variant is about 6MiB larger per architecture than default.
  */
-def jscFlavor = 'org.webkit:android-jsc:+'
+def jscFlavor = 'org.webkit:android-jsc-intl:+'
```

Comment:
+6 mb is quite a lot for such a simple functionality, I would really consider some JS polyfill instead